webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -70,6 +85,11 @@ const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
     [modalState, onClose],
   );
 
+  useLayoutEffect(() => {
```

Comment:
i understand why `useLayoutEffect` is here, but i think we can remove this by refactoring. 
the form is currently slow cuz we use react state, but we also use `form`. Its so much performant, so we should refactor later (not in the PR).